modules/extension/wms/src/main/java/org/geotools/ows/wms/WebMapServer.java

```diff
@@ -30,11 +30,7 @@
 import javax.swing.ImageIcon;
 import org.geotools.data.ResourceInfo;
 import org.geotools.data.ServiceInfo;
-import org.geotools.data.ows.AbstractOpenWebService;
-import org.geotools.data.ows.GetCapabilitiesRequest;
-import org.geotools.data.ows.GetCapabilitiesResponse;
-import org.geotools.data.ows.OperationType;
-import org.geotools.data.ows.Specification;
+import org.geotools.data.ows.*;
```

Comment:
Star imports are forbidden by our QA rules (as introduced by iant)

Reply:
Thanks I will turn that off, did not notice IDE doing it as I was mostly moving things around